vizro-core/src/vizro/models/__init__.py

```diff
@@ -22,7 +29,9 @@
     "Dropdown",
     "Filter",
     "Graph",
+    "NavItem",
     "Layout",
+    "NavBar",
```

Comment:
Could you ensure that this list stays alphabetically ordered? :)

Reply:
Corrected.